ratis-test/src/test/java/org/apache/ratis/datastream/TestDataStream.java

```diff
@@ -99,37 +104,81 @@ public WritableByteChannel getWritableByteChannel() {
       writeRequest = request;
       return CompletableFuture.completedFuture(stream);
     }
+
+    public int getByteWritten() {
```

Comment:
if you rename byteWritten to bytesWritten, you should also rename getByteWritten to getBytesWritten. But maybe, we should not rename byteWritten, it's unnecessary.

Reply:
I see. Changed back to `byteWritten`